internal/component/otelcol/exporter/loadbalancing/loadbalancing.go

```diff
@@ -93,6 +93,8 @@ func (args *Arguments) SetToDefault() {
 	}
 	args.DebugMetrics.SetToDefault()
 	args.Protocol.OTLP.SetToDefault()
+	args.Retry.SetToDefault()
+	args.Queue.SetToDefault()
```

Comment:
The fact that no default values were set must be an oversight. According to the Alloy documentation there should be default values for those two blocks.

Reply:
As @wildum mentioned in [another comment](https://github.com/grafana/alloy/pull/3489#discussion_r2079717638), this is actually intentional.